index.js

```diff
@@ -2,9 +2,12 @@ const getUrl = (str) => {
   const cleanedUrl = str.trim();
   const regex = /\[(.*?)\]\((.*?)\)/;
   const match = cleanedUrl.match(regex);
-  return match[2];
+  if (match) {
+    return match[2];
+  }
 };
```

Comment:
Nitpicking, but shouldn't we also return empty string/null in case of no matches?

Reply:
JS functions return `undefined` if we written nothing

Reply:
and it'll be filtered in the `Boolean` filter below it